libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/Schemas/Microsoft.Test.Script.schema

```diff
@@ -31,6 +31,12 @@
                 "$kind": "Microsoft.Test.IUserTokenMock"
             }
         },
+        "calledAsLeafSkill": {
+            "type": "boolean",
+            "title": "Called As Leaf Skill",
+            "description": "Enable mock this script to be called as a leaf skill (default is false)",
```

Comment:
>(default is false) [](start = 81, length = 18)

Should have period at the end like the other descriptions and leave off the default information--it is part of the schema.